charts/rancher-vsphere-cpi/Chart.yaml

```diff
@@ -5,10 +5,10 @@ annotations:
   catalog.cattle.io/namespace: kube-system
   catalog.cattle.io/os: linux
   catalog.cattle.io/permits-os: linux,windows
-  catalog.cattle.io/rancher-version: '>= 2.9.0-0'
+  catalog.cattle.io/rancher-version: '>= 2.11.0-0'
```

Comment:
Why do we need to make these incompatible with 2.11 only? We won't be updating the charts for 2.10 any longer?

Reply:
We will continue updating the chart for 2.9 and 2.10.  
I originally thought it would be tidy to update the value to the latest supported version and rely on the patch file in the rancher/charts repo to replace the value, but now I think it is better to not change it to avoid confusion as you just pointed out. 
The change was reverted.